src/Momento.Sdk.Incubating/Responses/CacheDictionaryFetchResponse.cs

```diff
@@ -5,56 +5,99 @@
-            return new Dictionary<byte[], byte[]>(
-                items.Select(kv => new KeyValuePair<byte[], byte[]>(kv.Field.ToByteArray(), kv.Value.ToByteArray())),
-                Utils.ByteArrayComparer);
-        });
+                return new Dictionary<byte[], byte[]>(
+                    items.Select(kv => new KeyValuePair<byte[], byte[]>(kv.Field.ToByteArray(), kv.Value.ToByteArray())),
+                    Utils.ByteArrayComparer);
+            });
+
+            _stringStringDictionary = new(() =>
+            {
+                return new Dictionary<string, string>(
+                    items.Select(kv => new KeyValuePair<string, string>(kv.Field.ToStringUtf8(), kv.Value.ToStringUtf8())));
+            });
+            _stringByteArrayDictionary = new(() =>
+            {
+                return new Dictionary<string, byte[]>(
+                    items.Select(kv => new KeyValuePair<string, byte[]>(kv.Field.ToStringUtf8(), kv.Value.ToByteArray())));
+            });
+        }
+
+        public Dictionary<byte[], byte[]>? ByteArrayByteArrayDictionary { get => _byteArrayByteArrayDictionary.Value; }
+
+        public Dictionary<string, string>? StringStringDictionary() => _stringStringDictionary.Value;
 
-        _stringStringDictionary = new(() =>
+        public Dictionary<string, byte[]>? StringByteArrayDictionary() => _stringByteArrayDictionary.Value;
+    }
+
+    public class Miss : CacheDictionaryFetchResponse
+    {
+        protected readonly Lazy<Dictionary<byte[], byte[]>?> _byteArrayByteArrayDictionary;
+        protected readonly Lazy<Dictionary<string, string>?> _stringStringDictionary;
+        protected readonly Lazy<Dictionary<string, byte[]>?> _stringByteArrayDictionary;
+        public Miss()
         {
-            if (items == null)
+            _byteArrayByteArrayDictionary = new(() =>
             {
                 return null;
-            }
-            return new Dictionary<string, string>(
-                items.Select(kv => new KeyValuePair<string, string>(kv.Field.ToStringUtf8(), kv.Value.ToStringUtf8())));
-        });
-        _stringByteArrayDictionary = new(() =>
-        {
-            if (items == null)
+            });
+
+            _stringStringDictionary = new(() =>
             {
                 return null;
-            }
-            return new Dictionary<string, byte[]>(
-                items.Select(kv => new KeyValuePair<string, byte[]>(kv.Field.ToStringUtf8(), kv.Value.ToByteArray())));
-        });
+            });
+            _stringByteArrayDictionary = new(() =>
+            {
+                return null;
+            });
+        }
+
+        public Dictionary<byte[], byte[]>? ByteArrayByteArrayDictionary { get => _byteArrayByteArrayDictionary.Value; }
+
+        public Dictionary<string, string>? StringStringDictionary() => _stringStringDictionary.Value;
+
+        public Dictionary<string, byte[]>? StringByteArrayDictionary() => _stringByteArrayDictionary.Value;
```

Comment:
We should be able to remove all of this. The `Miss` classes shouldn't have properties.

Reply:
Based on this test, we want to return null for Miss case: https://github.com/momentohq/client-sdk-dotnet-incubating/blob/c0c0705006ad924e65b4ad9868df245ca67ffa9b/tests/Integration/Momento.Sdk.Incubating.Tests/DictionaryTest.cs#L651

But I'd differ to @malandis 🙇‍♀️